typescript/src/schema/proto.ts

```diff
@@ -74,6 +75,7 @@ export interface MethodDescriptorProto
 
 export interface ServiceDescriptorProto
   extends protos.google.protobuf.IServiceDescriptorProto {
+  getAutoPopulatedFields: MethodDescriptorProto[];
```

Comment:
Is this field actually used anywhere?  I see the function `getAutoPopulatedFields` but I don't see if this added field is accessed anywhere.

Reply:
REmoved!